### PR TITLE
[WIP] Fixes

### LINKED ISSUE #23: cache permission issue
I run the containers and did make the composer install but the permissions for the cache files are not properly set and i get:

```
Failed to write cache file "/app/var/cache/dev/classes.php".
```

This problem is one of the main painpoints of using symfony with docker, imho, at least in a dev env.
I've been struggling with that for a while now and maybe we can find a solution here together.

You can read my struggle here: https://stackoverflow.com/questions/34949083/symfony-docker-permission-problems-for-cache-files.

### PR DESCRIPTION
**Pull request by mathewpeterson**
- [x] Resolve #23 
- [x] Change `date.timezone` to UTC
- [x] Update documentation with jwilder/nginx-proxy dependency

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ app:
     image: ubuntu:trusty
     volumes:
         - .:/app
+        - /app/app/cache
```
Comment (drgomesp) [inline docker-compose.yml]: Should be /app/var/cache

**Comment by drgomesp** [comment]
It should also resolve "Unable to create the cache directory (/app/var/cache/dev)"

**Comment by drgomesp** [comment]
It's still not working as expected.

```
Failed to write cache file "/app/var/cache/dev/classes.php".
```

I'm guessing the directories are owned by a different user than php-fpm is running on.

**Comment by mathewpeterson** [comment]
You will most likely need to run `docker-compose` stop, rm app, up -d

**Comment by drgomesp** [comment]
Ok, placing this:

```
RUN usermod -u 1000 www-data
```

In the docker/php/Dockerfile solves the problem.

_EDIT_: Latest solution didn't work for me.
_EDIT2_: It doesn't actually seem to solve the problem. Trying to figure out an alternative solution.

**Comment by drgomesp** [comment]
Ok, finally found it:

```
- ./var/cache:/app/var/cache
```

And

```
RUN usermod -u 1000 www-data
```

On the docker/php/Dockerfile

Can you try it out?

File: app/config/config_dev.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ monolog:
     handlers:
         main:
             type: stream
-            path: "%kernel.logs_dir%/%kernel.environment%.log"
+            path:   "/tmp/stdout"
```
Comment (drgomesp) [inline app/config/config_dev.yml:18]: This should be `php://STDOUT`

Comment (mathewpeterson) [inline app/config/config_dev.yml:18]: Sorry about that. There is a problem with Docker and the stdout stream that will not allow non-root users (like, www-data) from writing to it.  I've added an `entrypoint.sh` to handle this with more documentation.

**Comment by mathewpeterson** [comment]
@drgomesp . I don't see any need to mount the `var/cache` directory since this will cause permission issues between your host computer and your docker-machine. My thought would be to create a docker volume for `/app/var/cache` in the `app` container.

**Comment by drgomesp** [comment]
I understood your idea, but it doesn't work for some reason. Do you mind testing that?

**Pull request merged**